duo-client/src/test/java/com/duosecurity/client/HttpCanonRequestTest.java

```diff
@@ -57,6 +57,17 @@ private void assertCanonRequest(String expected_query_string,
                                  + expected_prefix
                                  + expected_query_string),
                                 actual);
+
+            try {
+                actual = h.canonRequest(date, 99);
+            }
+            catch (UnsupportedEncodingException e) {
+                Assert.fail(e.toString());
+                return;
+            }
+            Assert.assertEquals("failure - unsupport sig_version",
```

Comment:
can you add a comment here to make it clear that we expect the v1 output in this case? (essentially that we 'fall back' to v1 when we get an unsupported version)